_unittest/test_zEDB.py

```diff
@@ -195,9 +195,10 @@ def test_28_get_component_connections(self):
     def test_29_get_power_tree(self):
         OUTPUT_NET = "BST_V1P0_S0"
         GROUND_NETS = ["GND", "PGND"]
-        powertree_df, power_nets = self.edbapp.core_nets.get_powertree(OUTPUT_NET, GROUND_NETS)
-        assert len(powertree_df) > 0
-        assert len (power_nets) > 0
+        component_list, component_list_columns,net_group = self.edbapp.core_nets.get_powertree(OUTPUT_NET, GROUND_NETS)
+        assert len(component_list) > 0
+        assert len(component_list_columns) > 0
+        assert len (net_group) > 0
```

Comment:
It's a bit more pythonic to just assert that a list isn't empty.

```suggestion
        component_list, component_list_columns, net_group = self.edbapp.core_nets.get_powertree(OUTPUT_NET, GROUND_NETS)
        assert component_list
        assert component_list_columns
        assert net_group
```